setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+cd `dirname $0`
+
+# Create a virtual environment to run our code
```

Comment:
Comment doesn't really match code: this paragraph does not create a virtual environment. Try moving this to line 9ish.